tests/e2e/profile_test/profile_test.go

```diff
@@ -0,0 +1,43 @@
+package tests
+
+import (
+	"context"
+	"fmt"
+	"os"
+	"testing"
+
+	"github.com/HackIllinois/api/common/configloader"
+	"github.com/HackIllinois/api/tests/common"
+	"github.com/dghubble/sling"
+	"go.mongodb.org/mongo-driver/mongo"
+)
+
+var admin_client *sling.Sling
+var client *mongo.Client
+var profile_db_name string
+var unauthenticated_client *sling.Sling
+
+func TestMain(m *testing.M) {
+
+	cfg, err := configloader.Load(os.Getenv("HI_CONFIG"))
+
+	if err != nil {
+		fmt.Printf("ERROR: %v\n", err)
+		os.Exit(1)
+	}
+
+	admin_client = common.GetSlingClient("Admin")
+	unauthenticated_client = sling.New().Base("http://localhost:8000").Client(nil).Add("Authorization", "FAKE_TOKEN")
```

Comment:
Could we consider taking this `unauthenticated_client` out into `tests/common` so other tests can use it? Seems general enough to have a `common.GetSlingClient("Unauthenticated")`.